clink/app/src/loader/clink.bat

```diff
@@ -5,6 +5,19 @@
 set clink_profile_arg=
 set clink_quiet_arg=
 
+set clink_autorun=0
+set clink_parameters="%*"
+set clink_parameters_nospace="%clink_parameters: =_%"
+if not %clink_parameters_nospace:--autorun=%==x%clink_parameters_nospace% set clink_autorun=1
+
+:: quick exit if CLINK_NOAUTORUN is defined
+if defined CLINK_NOAUTORUN (
```

Comment:
The usage of `if defined` relies on Command Extensions, and causes `clink.bat` to fail whenever Command Extensions is disabled.